### PR TITLE
Add history command

### PR DESCRIPTION
**Pull request by Rinder5**

### CODE REVIEW

File: src/seedu/addressbook/ui/MainWindow.java [hunk 1]
```diff
@@ -84,6 +85,10 @@ public void displayResult(CommandResult result) {
         if(resultPersons.isPresent()) {
             display(resultPersons.get());
         }
+        final Optional<ArrayList<String>> resultCommands = result.getRelevantCommands();
+        if(resultCommands.isPresent()) {
```
Comment (louislai) [inline src/seedu/addressbook/ui/MainWindow.java:89]: I think you can leave a space between `if` and the brace, so `if(resultCommands.isPresent()) {` will be `if (resultCommands.isPresent()) {`

Comment (Rinder5) [reply src/seedu/addressbook/ui/MainWindow.java:89]: i followed the way they did it for the earlier if statement. if you want me to change it i can

**Review by Melvin-Tan — COMMENTED** [review]
Might want to add tests? 
Overall code is pretty neat!

**Review by Melvin-Tan — APPROVED** [review]

File: src/seedu/addressbook/ui/MainWindow.java (hunk 1 above)
Comment (louislai) [reply src/seedu/addressbook/ui/MainWindow.java:89]: LOL... I think I did spot this issue occasionally in their code base. Nvm it's okay this address book thing does not need to be perfect anw lol

**Review by louislai — APPROVED** [review]

File: src/seedu/addressbook/ui/MainWindow.java (hunk 1 above)
Comment (Melvin-Tan) [reply src/seedu/addressbook/ui/MainWindow.java:89]: Ya can leave it like that for now, but the coding standard for this module is to have a spacing there.
Even things like "for (int i=0; i<n; i++)" is not good, must be "for (int i = 0; i < n; i++)"

**Pull request merged**